CHANGELOG.md

```diff
@@ -4,14 +4,17 @@ Since v0.0.10, this Changelog is formatted according to the [Common Changelog][c
 
 ## UNRELEASED
 
+### Removed
+
+- ([#603](https://github.com/demos-europe/demosplan-ui/pull/603)) Remove components from ui, move it to Core. ([@ahmad-demos](https://github.com/@ahmad-demos))
```

Comment:
I'd suggest to list the removed components, but remove the mentioning of "core". We're writing this Changelog as the authors of demosplan-ui, not as the maintainers of demosplan-core.

```suggestion
- ([#603](https://github.com/demos-europe/demosplan-ui/pull/603)) Remove DpHeightLimit, DpSwitcher, and DpTextWrapper components. ([@ahmad-demos](https://github.com/@ahmad-demos))
```